src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,15 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie, MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {
+      movies.map(movie => <MovieCard key={movie.imdbId} movie={movie} />)
+    }
```

Comment:
Try to use more readable layout.
```suggestion
    {movies.map(movie => (
      <MovieCard
        key={movie.imdbId}
        movie={movie} 
      />)}
```